src/Sulu/Bundle/MediaBundle/Resources/js/stores/CollectionStore/CollectionStore.js

```diff
@@ -8,6 +8,7 @@ const COLLECTIONS_RESOURCE_KEY = 'collections';
 export default class CollectionStore {
     @observable loading: boolean = false;
     @observable collection: Collection = {
+        id: null,
```

Comment:
I think I would have gone here for separate variables... Accessing `CollectionStore.collection` seems weird... Or is that necessary in some way?

Reply:
Will be changed in an upcoming PR